pandas/tests/series/test_datetime_values.py

```diff
@@ -682,6 +682,7 @@ def test_setitem_with_different_tz(self):
             [[pd.NaT], [[np.NaN, np.NaN, np.NaN]]],
             [["2019-12-31", "2019-12-29"], [[2020, 1, 2], [2019, 52, 7]]],
             [["2010-01-01", pd.NaT], [[2009, 53, 5], [np.NaN, np.NaN, np.NaN]]],
+            [["2016-01-08"], [[2016, 1, 5]]],  # see GH#36032
```

Comment:
can you add some of the working cases from the original issue (that work)

Reply:
done

also tested with this locally
```python
import pandas as pd
dates = pd.date_range('1677-09-22', '2262-04-10', freq='D')
pd_iso = dates.isocalendar()
py_dates_iso = [date.isocalendar() for date in dates.to_pydatetime()]
py_iso = pd.DataFrame(py_dates_iso, columns=['year', 'week', 'day'], index=dates, dtype='UInt32')
assert pd_iso.equals(py_iso)
```